src/anet.c

```diff
@@ -194,6 +194,7 @@ int anetKeepAlive(char *err, int fd, int interval)
     }
 
     intvl = idle/3;
+    if (intvl < 10) intvl = 10; // kernel expects at least 10 seconds
```

Comment:
Use `/* */` comments.

```suggestion
    if (intvl < 10) intvl = 10; /* kernel expects at least 10 seconds */
```

On the linked page, at first I didn't find this when searching for "10" because it is written in as "ten seconds". So this seems correct. Quote from the [docs](https://docs.oracle.com/cd/E88353_01/html/E37851/tcp-4p.html):

> The second implementation is activated if socket option TCP_KEEPINTVL and/or TCP_KEEPCNT are set. The time between each consequent probes is set by TCP_KEEPINTVL in seconds. The minimum value is ten seconds. The maximum is ten days, while the default is two hours. The TCP connection will be aborted after certain amount of probes, which is set by TCP_KEEPCNT, without receiving response.